content/en/docs/reference/glossary/webhook.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Webhook
+id: webhook
+date: 2019-02-13
+full_link: /docs/reference/access-authn-authz/webhook/
+short_description: >
+  A way for one component to notify another component in real time, via HTTP.
+
+aka:
+tags:
+- networking
+---
+ A notification message sent from one component to another component's unique URL,
```

Comment:
```suggestion
 A notification message sent from one web-based service to another service's unique URI,
```

Reply:
All the Kubernetes things that send webhooks are also HTTP servers, I think. Is this generally true?
Some things that can send webhooks, and aren't necessarily “web-based”:
- an uninterruptible power supply
- a vending machine that monitors its stock levels
- my smartphone

I think it's fair to call the _receiving_ side “web-based”, even if handling webhooks is a small part of its overall behaviour.